test/util/queryElements.js

```diff
@@ -6,14 +6,18 @@ const { By } = require('selenium-webdriver');
  * @param {ExecutionContext} t - Test execution context
  * @param {String} selector - CSS selector string
  * @param {Element} context - Element to query within, defaulting to t.context.session
+ * @param {Boolean} noTest - When true, allow empty results
  *
  * @returns {Promise} Resolves to array of elements
  */
-module.exports = async function queryElements(t, selector, context) {
+module.exports = async function queryElements(t, selector, context, noTest) {
```

Comment:
I think we should probably add an "options" object with `{ allowNoResults : true }`. The pattern you used is called a "boolean trap" and it's very hard to understand when reading the code.

Just a heads up: We also created this function `queryElements` to make sure that we don't have false positives in tests, especially when we are testing in a loop. If we are looping over elements returned by `queryElements` to test each element, but the css selector was incorrect, then we never enter the loop and the test false passes!

Reply:
@spectranaut 
I don't need the change anymore, no need for no result option.